src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ReadAndUpdateOperation.java

```diff
@@ -16,8 +16,10 @@
 import io.stargate.sgv2.jsonapi.service.shredding.model.WritableShreddedDocument;
 import io.stargate.sgv2.jsonapi.service.updater.DocumentUpdater;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Supplier;
-import java.util.stream.Stream;
 
 public record ReadAndUpdateOperation(
```

Comment:
I think it'd be good to have Javadoc here to explain which operations this is used for: I am guessing it is for update and delete ones but would be good to clarify.

Reply:
This operation method is used for 3 commands findOneAndUpdate, updateOne and updateMany, Will add javadoc for this